example/lib/main.dart

```diff
@@ -62,7 +61,7 @@ class _MyAppState extends State<MyApp> {
 
     // Initialize AmplifyFlutter
     try {
-      await Amplify.configure(amplifyconfig);
+      await Amplify.configure('amplifyconfig');
```

Comment:
Same as above. Let's keep this as is.
```suggestion
      await Amplify.configure(amplifyconfig);
```